(DOCSP-24458): Convert existing SwiftUI examples to new test app and docs structure (#2112)
mongodb/docs-realm: source/sdk/swift/swiftui/filter-data.txt

=====================
Filter Data - SwiftUI
=====================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Search a Realm Collection
-------------------------

.. versionadded:: 10.19.0

The Realm Swift SDK allows you to extend :apple:`.searchable 
<swiftui/form/searchable(text:placement:prompt:)-6royb>`. When you use 
:swift-sdk:`ObservedResults <Structs/ObservedResults.html>` to query a realm,

> you can specify collection and keypath in the result set to denote it is

Reviewer: What are collection and keypath supposed to be?

> you can specify collection and keypath in the result set to denote it is

Reviewer: Ultra-nitpick: For some reason the word "denote" really sticks out to me as rare:
```suggestion
you can specify collection and keypath in the result set to mark it as
```
